README.md

```diff
@@ -156,6 +160,39 @@ See https://ariejan.net/2010/06/10/cherry-picking-specific-commits-from-another-
     - Click "Create pull request"
     - Add a description of the pull request and start the pull request description with the story or epic ID, `CI-XXX`
 
+## Start Rails server
+
+To start a Rails server with React, you need to  use the command:
+
+`rake assets:precompile`
```

Comment:
Please specify how often to run `rake assets:precompile`. Is it every time you start the server, or maybe only after significant changes to react have been done or pulled from Github?